.github/workflows/conformance.yaml

```diff
@@ -0,0 +1,48 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# Github action job to test core java library features on
+# downstream client libraries before they are released.
+on:
+  push:
+    paths:
+      - 'bigtable/**'
+    branches:
+    - main
+  pull_request:
+    paths:
+      - 'bigtable/**'
+name: conformance
+jobs:
+  conformance:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        go: [ '1.19', '1.20', '1.21' ]
```

Comment:
Why have a matrix strategy if it can only be run with one Go version? Just install that version specifically in the action and simplify the script code.

Reply:
I think the intent here was to have a particular golang version running for the test proxy server https://github.com/googleapis/google-cloud-go/pull/8658/files#diff-25791c2e81a435384b6d9fe6d3386af6ed532964e933261c53fbe00ff7bbc3e3R37 so that we can test the test proxy server on multiple golang versions. Currently, the way this is written is the test runner, which calls the test proxy server, has a fixed Golang version.

In the node client multiple node versions are tested because streaming behavior can vary by node version. I wonder if the same is true for Golang.

Reply:
> streaming behavior can vary by node version. I wonder if the same is true for Golang.

I'm not sure, but the explanation sounds reasonable. I think it would be useful to have some/all of this detail documented in either a readme for or in comments.

Reply:
Added